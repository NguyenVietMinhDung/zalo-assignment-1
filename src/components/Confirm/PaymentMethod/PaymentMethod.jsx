import React from 'react';
import { Container, Row, Col } from 'react-grid-system';
import './PaymentMethod.scss';
import zalopay from '../../../assets/images/ZALOPAY.png';

const PaymentMethod = () => (
  <Container>
    <Row className="payment-method">
      <Col xs={12}>
        <div className="payment-method__title">
          <span>Chỉ thanh toán bằng</span>&nbsp;
          <img src={zalopay} alt="" />
        </div>
      </Col>
      <Col xs={12}>
        <div className="payment-method__btn-container">
          <div className="payment-method__btn">
            Thanh toán 107.000đ
          </div>
        </div>
      </Col>
    </Row>
  </Container>
);

export default PaymentMethod;
