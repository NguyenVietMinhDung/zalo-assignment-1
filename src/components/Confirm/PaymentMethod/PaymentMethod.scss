@import '../../../styles/variables';

.payment-method {
	display: flex;

	flex-direction: column;

	margin-bottom: 8px;

	padding-top: 15px;
	padding-bottom: 15px;

	background-color: white;

	&__title {
		display: flex;

		font-size: 12px;
		font-weight: 500;
		align-items: center;
		justify-content: center;

		span {
			opacity: 0.5;
		}

		img {
			width: 40px;
		}
	}

	&__btn-container {
		display: flex;

		padding: 10px 0;
		align-items: center;
		justify-content: center;
	}

	&__btn {
		width: 400px;
		max-width: 100vw;
		padding: 10px 20px;

		cursor: pointer;
		text-align: center;

		color: white;
		border-radius: 50px;
		background-color: $light-blue;

		font-weight: 500;
	}
}
