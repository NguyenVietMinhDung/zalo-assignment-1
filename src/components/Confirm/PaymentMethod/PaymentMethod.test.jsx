import React from 'react';
import { shallow } from 'enzyme';
import PaymentMethod from './PaymentMethod';

describe('<PaymentMethod />', () => {
  test('renders correctly', () => {
    const component = shallow(<PaymentMethod />);
    expect(component).toMatchSnapshot();
  });
});
