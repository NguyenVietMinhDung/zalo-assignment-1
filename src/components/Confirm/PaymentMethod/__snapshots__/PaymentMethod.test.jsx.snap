// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PaymentMethod /> renders correctly 1`] = `
<Container
  component="div"
  fluid={false}
  lg={false}
  md={false}
  sm={false}
  style={Object {}}
  xl={false}
  xs={false}
  xxl={false}
>
  <Row
    align="normal"
    className="payment-method"
    component="div"
    debug={false}
    gutterWidth={null}
    justify="start"
    nogutter={false}
    nowrap={false}
    style={Object {}}
  >
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <div
        className="payment-method__title"
      >
        <span>
          Chỉ thanh toán bằng
        </span>
         
        <img
          alt=""
          src="test-file-stub"
        />
      </div>
    </Col>
    <Col
      component="div"
      debug={false}
      lg={null}
      md={null}
      offset={Object {}}
      pull={Object {}}
      push={Object {}}
      sm={null}
      style={Object {}}
      width={null}
      xl={null}
      xs={12}
      xxl={null}
    >
      <div
        className="payment-method__btn-container"
      >
        <div
          className="payment-method__btn"
        >
          Thanh toán 107.000đ
        </div>
      </div>
    </Col>
  </Row>
</Container>
`;
